src/options/parallel_options.toml

```diff
@@ -0,0 +1,65 @@
+id     = "PARALLEL"
+name   = "Parallel"
+
+[[option]]
+  name       = "computePartitions"
+  category   = "expert"
+  long       = "compute-partitions=N"
+  type       = "uint64_t"
+  default    = "0"
+  help       = "make n partitions. n <2 disables computing partitions entirely"
+
+[[option]]
+  name       = "checksBeforePartitioning"
+  category   = "expert"
+  long       = "checks-before-partition=N"
+  type       = "uint64_t"
+  default    = "1"
+  help       = "number of standard or full effort checks until partitioning"
+
+[[option]]
+  name       = "partitionConflictSize"
+  category   = "expert"
+  long       = "conflict-size=N"
```

Comment:
Can we change the name here to partition-conflict-size?

Reply:
Also, I think 0 means some default involving log2 is chosen, say that in the help here.